src/Analyzers/CSharp/Tests/UseUTF8StringLiteral/UseUTF8StringLiteralTests.cs

```diff
@@ -0,0 +1,596 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.UseUTF8StringLiteral;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.UseUTF8StringLiteral
+{
+    using VerifyCS = CSharpCodeFixVerifier<
+        UseUTF8StringLiteralDiagnosticAnalyzer,
+        UseUTF8StringLiteralCodeFixProvider>;
+
+    public class UseUTF8StringLiteralTests
```

Comment:
Consider testing something like `public void M(params byte[] bytes) { }` with call site both with explicit array creation (`M(new byte[] { some bytes here })` and implicit array creation `M(some bytes here)`